FAQ EDITS (#211)
gbnewby/gutenbergsite: site/help/faq.md

### Tell me about Project Gutenberg's longevity

> Project Gutenberg is the original, and oldest, eBook project on the Internet, founded in 1971. It is one of the oldest online content providers in the world that is still operating. It was established before the World Wide Web and even the Internet itself. Project Gutenberg started at one of the 15 ARPANET sites that later formed the core of the Internet.

Reviewer: same internet, different name. This added comment distracts from the message

> Project Gutenberg is the original, and oldest, eBook project on the Internet, founded in 1971. It is one of the oldest online content providers in the world that is still operating. It was established before the World Wide Web and even the Internet itself. Project Gutenberg started at one of the 15 ARPANET sites that later formed the core of the Internet.

Reviewer: ```suggestion
Project Gutenberg is the original, and oldest, eBook project on the Internet, founded in 1971. It is one of the oldest online content providers in the world that is still operating. 
```

Help improve our eBooks! If you find an error, or something missing - or perhaps you have a suggestion for our website - please [get in touch](/about/contact_information.html).

> Spread the word! Invite your friends and colleagues to visit Project Gutenberg at gutenberg.org. When you find something you like, share it via email, word-of-mouth, or on social media. If you want to redistribute the books yourself, see the [Terms of Use](/policy/terms_of_use.html) for details and limitations.

Author: Inviting people to share the books but then immediately pointing to "terms of use" is off-putting. I am trying to make a separation between the individual's sharing action and the legalese associated with using the PG trademark that makes up so much of the ToS.

> Spread the word! Invite your friends and colleagues to visit Project Gutenberg at gutenberg.org. When you find something you like, share it via email, word-of-mouth, or on social media. If you want to redistribute the books yourself, see the [Terms of Use](/policy/terms_of_use.html) for details and limitations.

Reviewer: ```suggestion
Spread the word! Invite your friends and colleagues to visit Project Gutenberg. When you find something you like, share it via email, word-of-mouth, or on social media. Even [commercial use](/policy/terms_of_use.html) is possible.
```

### What other things has Project Gutenberg published in the past?

We have published music files, in MIDI and MUS formats. We have published the Human Genome. We have published pictures of the prehistoric cave painting from the south of France. We have published video files and audio files, including tracks by award-winning singer/songwriter Janis Ian and readings from public domain books.

Today the primary focus is to distribute literary works that were previously published and are now in the public domain. The [collection development](/policy/collection_development.html) policy has details.

In 2011 we began the Project Gutenberg Self Publishing Portal to focus on distribution of new works. Since then the self-publishing portal has released over 20,000 titles. [self.gutenberg.org](http://self.gutenberg.org/)

### How does Project Gutenberg choose books to publish?

Project Gutenberg, as such, does not choose books to publish. There is no central list of works that volunteers are asked to work on. Individual volunteers choose and produce eBooks from printed books according to their own tastes and values, and the availability of a suitable printed book for digitization.

> 

Reviewer: Mention distributed proofreaders?

Author: That wasn't originally in this part of the FAQ so it didn't cross my mind to expand it in this PR. I will add that to my notes for later revisions, after this update.

Reviewer: ok, I'll put something in.
